rectools/models/rank.py

```diff
@@ -252,6 +259,7 @@ def rank(  # pylint: disable=too-many-branches
                 filter_query_items=filter_query_items,
             )
         else:
+            check_blas_config()
```

Comment:
Please add a comment why do we need this

Reply:
I was pretty sure that it affects implicit ranking speed as well as it affects implicit training speed. But now I've checked again and it's not the case. Removed.